ActindoWorkFlow/UserInterface/08_CoreActions.md

```diff
@@ -0,0 +1,251 @@
+  - **in**    
+    It is a match if the input value is included within the defined values. For this operator, it is possible to enter an array in the *Value* field.
+  - **notIn**   
+    It is a match if the input value is not included within the defined values. For this operator, it is possible to enter an array in the *Value* field.
+  - **regexp**   
+    It is a match if the input value equals the regular expression entered in the *Value* field.    
+    For detailed information about regular expressions, see https://regex101.com/.
+
+
+- *Value*   
+  Enter the value to be compared with the input value. The value must be valid JSON and is type strict, for example a string must be specified as a string (**\"example\"**), an integer as an integer (**123**), and so on.
+
+[comment]: <> (Weitere operatoren geplant: <=, <, >=, >)
+
+
+
+## Start subprocess
+
+![Start subprocess](../../Assets/Screenshots/ActindoWorkFlow/Workflows/CoreActions/StartSubprocess.png "[Start subprocess]")
+
+The *Start subprocess* core action is used to start a different subprocess within the currently running process.       
+The data runs via the *p* input port into the workflow action and is output via the *p* output port. However, the action does not wait until the subprocess has been finished, but outputs the data after having started the subprocess. The input data is also the data put in the start place of the subprocess.   
+The subprocess to be started is defined in the configuration.
+
+### Configuration
+
+- *Workflow Key*   
+  Enter the workflow key of the workflow that will be started as a subprocess.
+
+
+
+## Switch case action
+
+![Switch case action](../../Assets/Screenshots/ActindoWorkFlow/Workflows/CoreActions/SwitchCaseAction.png "[Switch case action]")
+
+The *Switch case action* core action is used to compare the input value with up to 6 criteria and output this value via a different branch for each case. Additionally, this core action enables to output the input value via a separate branch if the input value does not match any of the specified cases or to output the input value without any comparison by a separate branch.    
+The data runs via the *in* input port into the workflow action and can be output via each connected output port. If the *origin* output port is connected, the input value is always output via this port without any further action. Additionally, if one or several of the *case1* to *case6* output ports are connected, the input value is compared with the criterion specified in the respective case and output via the output port of the case where the criterion matches the input value. If no criterion of the cases matches the input value, the input value is output via the *default* output port, or, if the *default* output port is not connected, the action fails.    
+The criteria to be compared with are defined in the configuration. Further, you can set the comparison to be stopped after a match.
```

Comment:
```suggestion
The criteria to be compared with are defined in the configuration. Further, you can set the comparison to be stopped after a match. Otherwise all remaining cases are also evaluated.
```